recipes/schist/LICENSE

```diff
@@ -0,0 +1,11 @@
+Copyright 2020 Davide Cittaro
```

Comment:
No need to include this file manually since it's included in the GitHub tarball.